dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/startup/SchedulerStart.java

```diff
@@ -48,29 +48,44 @@
 
 /**
  *
- * Reschedule old jobs and execute jobs which were scheduled when the server was not running.
+ * Reschedule old jobs and execute jobs which were scheduled when the server was
+ * not running.
  *
  * @author Henning Håkonsen
  */
-public class SchedulerStart
-    extends AbstractStartupRoutine
+public class SchedulerStart extends AbstractStartupRoutine
```

Comment:
You should set the leaderJob here as well. Only doing it in the TableAlteror will not work if we create new Jobs here.

Reply:
Got it.